### ISSUE TITLE
A11y: Home page

### ISSUE THREAD

**Issue by polibb** [cherrytwist/Coordination#1581]
# Background
After going through the [Alkemio pages, sections, and modals](https://stichtingcherrytwist-my.sharepoint.com/:x:/g/personal/polina_alkem_io/EajcsuISFpJAg1FIbPM2F3MBGwpwyvh5qfGL0-UUmzvbxw?e=dQ0tPL) to fix as many issues as possible of the 4.1 Criteria from the WCAG 2.1, now creating issues for each page/section that follows this initiative. This will be a better structured approach to cover the highest priority uses of Alkemio for our users first, fixing all WCAG 2.1 Criteria found. The Criteria with highest priority remain the ones listed in this epic in separate sub-issues.

This is a task for the Home page, its dialogs, and its optional views depending on user's role.

# Acceptance Criteria

[Excel file with a non-exhaustive list of pages and dialogs in Alkem.io](https://stichtingcherrytwist-my.sharepoint.com/:x:/g/personal/polina_alkem_io/EajcsuISFpJAg1FIbPM2F3MBGwpwyvh5qfGL0-UUmzvbxw?e=4e0IaS)

As proven by [Chrome extension for automatic checks by Deque](https://www.deque.com/axe/browser-extensions/):

- [x] 	1.1 Text Alternatives - mainly 1.1.1
- [x] 	1.3 Adaptable - mainly 1.3.1, 1.3.5
- [x] 	1.4 Distinguishable - mainly 1.4.3, 1.4.4, 1.4.11, 1.4.12, 1.4.13
- [x] 	3.1 Readable - mainly 3.1.2
- [x] 	3.2 Predictable - mainly 3.2.3, 3.2.4
- [x] 	3.3 Input Assistance - mainly 3.3.1, 3.3.3
- [x] 	4.1 Compatible - mainly 4.1.2, 4.1.3

Manual testing: 

- [x]         2. Operable - Keyboard accessible:
- [x] 	2.1 Keyboard Accessible - mainly 2.1.1, 2.1.2
- [x] 	2.4 Navigable - mainly 2.4.1, 2.4.2, 2.4.7
- [x] 	2.5 Input Modalities - mainly 2.5.3, 2.5.5

https://github.com/user-attachments/assets/822f36dd-c00c-4066-92c7-9cc56bbc65a1

https://github.com/user-attachments/assets/c28bfd5b-ce75-4788-83a5-669187ac8e9f

# Context

[Excel file](https://stichtingcherrytwist.sharepoint.com/:x:/s/Product/EawxIsWzBaRHt8K2E1tUwmsBzd93mWRRiImOP2M0muxujQ?e=eYV2Qr)

[W3C website](https://www.w3.org/TR/WCAG21/)

[Excel file with a non-exhaustive list of pages and dialogs in Alkem.io](https://stichtingcherrytwist-my.sharepoint.com/:x:/g/personal/polina_alkem_io/EajcsuISFpJAg1FIbPM2F3MBGwpwyvh5qfGL0-UUmzvbxw?e=4e0IaS)

**Comment by bobbykolev** [comment]
The screen recordings in the description 😍 👏

**Comment by bobbykolev** [comment]
QA notes:

- mobile layout - attr for disabling user zoom was removed;
- now all dialogs should be dismissible with ESC;
- check for regressions the Avatar;

**Comment by polibb** [comment]
- now all dialogs should be dismissible with ESC can be solving also this open issue: https://app.zenhub.com/workspaces/alkemio-development-5ecb98b262ebd9f4aec4194c/issues/gh/alkem-io/client-web/5221 (put in QA now)

**Comment by polibb** [comment]
QA notes (this work is more "hidden" - screen readers and keyboard users care about it):
- all dialogs have a title attached to their label (aria-labelledby on dialogs, aria-label on title)
- no empty h1, h2, h3, etc. tags
- everything you can do with a mouse you can do with a keyboard :) no keyboard-traps! When you go somewhere you can leave

**Comment by Comoque1** [comment]
Tested on ACC env only for regressions on different browser and OS configurations
Only one issue identified and is currently being addressed: the margin between rows:
<img width="464" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNVJHQ3c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--08de183433ff13b129c172075f6b0d62c3426118/image.png" alt="image.png" />
